sa.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-07/schema
+title: GA4GH-SA-Definitions
+type: object
+definitions:
+  SequenceFeature:
+    properties:
+      id:
+        type: string
+      location:
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/main/schema/vrs.yaml#definitions/Location"
+      seqOntologyRef:
+        type: string
+    required:
+    - id
+    - location
+    - seqOntologyRef
+    oneOf:
+    - "$ref": "#/definitions/Gene"
+    - "$ref": "#/definitions/Transcript"
+    - "$ref": "#/definitions/TranscriptRegion"
+  Gene:
+    properties:
+      transcripts:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "#/definitions/Transcript"
+      strand:
+        type: string
+        enum:
```

Comment:
Strand is likely a reusable concept. Should be yanked out, but also has this been covered in VA? If not make sure this is the right way to encode

Reply:
@lstein's [2020 GFF3 spec](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md) encodes strand this way.